Add project tenets (#86)
aws/eks-distro: docs/contents/index.md

See the [Partners](community/partners.md) page for links to third-party methods for
installing EKS-D.

## Project Tenets

The tenets of the EKS Distro (EKS-D) project are:

1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere

> 2. **Simple**: make Kubernetes operations, administration, and management simple and boring

Reviewer: ```suggestion
2. **Simple**: Make Kubernetes operations, administration, and management simple and boring
```

> 2. **Simple**: Make Kubernetes operations, administration, and management simple and boring

Reviewer 2: Focus on EKS-D tenets here:
> Simple: Make using a Kubernetes distribution simple and boring (reliable and secure).

> 3. **Opinionated Modularity**: provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out

Reviewer: ```suggestion
3. **Opinionated Modularity**: Provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
```

> 3. **Opinionated Modularity**: Provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out

Reviewer 2: Again, more eks-d focused: s/tooling/components/.

> 4. **Open**: provide open source tooling backed, validated and supported by Amazon

Reviewer: ```suggestion
4. **Open**: Provide open source tooling backed, validated, and maintained by Amazon
```

> 4. **Open**: provide open source tooling backed, validated and maintained by Amazon

Reviewer: nit
```suggestion
4. **Open**: Provide open source tooling backed, validated and maintained by Amazon
```

> 5. **Ubiquitous**: provide support for the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets

Reviewer: ```suggestion
5. **Ubiquitous**: Enable customers to run Kubernetes on the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets
```

> 5. **Ubiquitous**: Enable customers to run Kubernetes on the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets

Reviewer 2: EKS-D focused: 
> Ubiquitous: Enable customers and partners to integrate a Kubernetes distribution in the most common tooling (Kubernetes installers and distributions, infrastructure as code, and more).

> 6. **Stand Alone**: provided for use anywhere without AWS dependencies (on-premises air-gapped)

Reviewer: ```suggestion
6. **Stand Alone**: Provided for use anywhere without AWS dependencies (on-premises air-gapped)
```

> 6. **Stand Alone**: Provided for use anywhere without AWS dependencies (on-premises air-gapped)

Reviewer 2: drop on-prem and air gapped distinction here.

> 7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services

Reviewer 2: for the eks-d focused tentets - I think this is not relevant.

Reviewer: Lets keep it, we do include AWS-specific components (authenticator) and could include the KMS encryption provider, cloud provider, or others in the future

Reviewer 2: sgtm

> 7. **Better with AWS**: enable AWS customers to enlist AWS Enterprise Support and allow them easily adopt additional AWS services

Reviewer: I like the title, lets change the wording
```suggestion
7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services
8. ```

Reviewer 2: lgtm
